### PR TITLE
installer: provide a SUSE package

### PR DESCRIPTION
**Pull request by chouquette**
### What does this PR do?

This PR adds a SUSE package for the datadog installer

### Motivation

Supporting the same set of distribution as the datadog agent

### Additional Notes

This is based on top of https://github.com/DataDog/datadog-agent/pull/24568

### CODE REVIEW

**Review by spencergilbert — APPROVED** [review]

File: test/new-e2e/tests/updater/linux_test.go [hunk 1]
```diff
@@ -70,6 +70,10 @@ func TestDebianX86(t *testing.T) {
 	runTest(t, "dpkg", os.AMD64Arch, os.UbuntuDefault)
 }
 
+func TestSuse(t *testing.T) {
+	runTest(t, "rpm", os.AMD64Arch, os.SuseDefault)
+}
```
Comment (raphaelgavache) [inline test/new-e2e/tests/updater/linux_test.go:75]: adding a arm64 too would be great to ensure 100% coverage of each package

Comment (chouquette) [reply test/new-e2e/tests/updater/linux_test.go:75]: indeed, done

**Review by pducolin — APPROVED** [review]

File: test/new-e2e/tests/installer/linux_test.go [hunk 2]
```diff
@@ -79,6 +79,14 @@ func TestDebianX86(t *testing.T) {
 	runTest(t, "dpkg", os.AMD64Arch, os.DebianDefault, true)
 }
 
+func TestSuseX86(t *testing.T) {
+	runTest(t, "rpm", os.AMD64Arch, os.SuseDefault, false)
```
Comment (raphaelgavache) [inline test/new-e2e/tests/installer/linux_test.go:83]: nice!
Unfortunately you should disable those tests, as all tests are now [disabled temporarily due to a breaking on going changes: ](https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/tests/installer/linux_test.go#L58-L80).
They'll be enabled back in the next day, and I'll remove the skip from Suse when it's time

**Review by raphaelgavache — APPROVED** [review]

**Pull request merged**